File: CWM/FoodRecipe/app/src/main/java/br/com/daniel/ramos/foodrecipe/request/responses/RecipeSearchResponse.java
package br.com.daniel.ramos.foodrecipe.request.responses;

import com.google.gson.annotations.Expose;
import com.google.gson.annotations.SerializedName;

import java.util.List;

import br.com.daniel.ramos.foodrecipe.models.Recipe;

public class RecipeSearchResponse {

    @SerializedName("count")
    @Expose()
    private int count;

    @SerializedName("recipes")
    @Expose()
    private List<Recipe> recipes;

    public int getCount() {
        return count;
    }

    public List<Recipe> getRecipes() {
        return recipes;
    }

    @Override
    public String toString() {
        return "RecipeSearchResponse{" +
                "count=" + count +
                ", recipes=" + recipes +
                '}';
    }
}

File: CWM/LearningJetpackCompose/app/src/main/java/br/com/daniel/ramos/learningjetpackcompose/network/response/RecipeSearchResponse.kt
package br.com.daniel.ramos.learningjetpackcompose.network.response

import br.com.daniel.ramos.learningjetpackcompose.network.model.RecipeDto
import com.google.gson.annotations.SerializedName

data class RecipeSearchResponse (
    @SerializedName("count")
    var count: Int,
    @SerializedName("results")
    var recipes: List<RecipeDto>,
    )